daemon/access.go

```diff
@@ -246,3 +247,34 @@ func (ac interfaceAuthenticatedAccess) CheckAccess(d *Daemon, r *http.Request, u
 
 	return Unauthorized("access denied")
 }
+
```

Comment:
Can you please add a comment here describing `noticeReadAccess`? Something like:

```go
// noticeReadAccess allows access to change-update notices for snaps
// which plug the snap-refresh-observe interface and otherwise behaves
// like openAccess (ie. allows access from snapd.socket only)
```

Reply:
thank you, updated